mne/epochs.py

```diff
@@ -3523,8 +3555,15 @@ def _read_one_epoch_file(f, tree, preload):
 
         # read in the Annotations if they exist
         annotations = _read_annotations_fif(fid, tree)
-        events, mappings = _read_events_fif(fid, tree)
-
+        try:
+            events, mappings = _read_events_fif(fid, tree)
+        except ValueError as e:
+            # Allow reading empty epochs (ToDo: Maybe not anymore in the future)
+            if str(e) == "Could not find any events":
+                events = np.empty((0, 3), dtype=np.int32)
+                mappings = dict()
+            else:
+                raise e
```

Comment:
A bare `raise` behaves better than `raise e` (it makes it omit the current line in the traceback)
```suggestion
                raise
```